casbin/management_enforcer.py

```diff
@@ -151,6 +151,20 @@ def update_named_policies(self, ptype, old_rules, new_rules):
         """updates authorization rules from the current named policy."""
         return self._update_policies("p", ptype, old_rules, new_rules)
 
+    def update_filtered_policies(self, new_rules, field_index, *field_values):
+        """updates all the filtered authorization rules to new rules."""
+        return self.update_filtered_named_policies(
+            "p", new_rules, field_index, *field_values
+        )
+
+    def update_filtered_named_policies(
+        self, ptype, new_rules, field_index, *field_values
+    ):
+        """updates all the filtered authorization rules to new rules."""
+        return self._update_filtered_policies(
+            "p", ptype, new_rules, field_index, *field_values
+        )
```

Comment:
@Shivansh-yadav13  Please add unit tests.

Reply:
@leeqvip I have not done much testing in python but on adding the test and testing, this is what I get can you tell what could be wrong here?
```
.....E.....E
======================================================================
ERROR: test_update_filtered_policies (__main__.TestManagementApi)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/shivansh/dev/os/pycasbin/tests/test_management_api.py", line 317, in test_update_filtered_policies
    e.update_filtered_policies([["data2_admin", "data3", "read"], ["data2_admin", "data3", "write"]], 0, "data2_admin")
AttributeError: 'Enforcer' object has no attribute 'update_filtered_policies'

======================================================================
ERROR: test_update_filtered_policies (__main__.TestManagementApiSynced)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/shivansh/dev/os/pycasbin/tests/test_management_api.py", line 317, in test_update_filtered_policies
    e.update_filtered_policies([["data2_admin", "data3", "read"], ["data2_admin", "data3", "write"]], 0, "data2_admin")
AttributeError: 'SyncedEnforcer' object has no attribute 'update_filtered_policies'

----------------------------------------------------------------------
Ran 12 tests in 0.009s

FAILED (errors=2)
```

Reply:
@leeqvip I have not added tests because we are not adding it for the file adapter as discussed below.